.github/workflows/run-test-sh.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: ${{ github.event_name == 'push' }}
-          load: true
+          load: ${{ github.event_name != 'push' }}
```

Comment:
Why is this line needed at all ?

Reply:
I think it's something that the previous PR started doing but didn't finish. Docker buildx (or BuildKit) has different output targets - `registry` when `push: true` and `docker`, when `load: true`. If we don't export the image in `docker` format we want be able to do do `docker run` on it, and so we won't be able to run the test suite. We have 2 options - either use non-buildx docker, or do what we do now - run test suite only for merge requests and not for push events.

Reply:
> Why is this line needed at all ?

`load: true` makes the built image accessible for manual `docker` usage in `test.sh`

> We have 2 options - either use non-buildx docker, or do what we do now - run test suite only for merge requests and not for push events.

Can we add pushing via `docker/build-push-action@v2` as a last  step? Or will it rebould the entire image?